packages/library/src/common/type.library.ts

```
@@ -3,22 +3,39 @@ import { $Extensible, $Target } from '#library/symbol.library.js';
+ * before calling getType() to ensure custom types are correctly identified.
  */
 export const getType = (obj?: any, ...instances: Instance[]) => {
 	const type = protoType(obj) as Type;
 
 	switch (true) {
 		case type === 'Object':
 			const name = isArrayLike(obj)
-				? 'ArrayLike'																				// special case Object: ArrayLike
-				: obj.constructor?.name ?? 'Object'									// some Objects do not have a constructor method
+				? 'ArrayLike'																			// special case Object: ArrayLike
+				: obj.constructor?.name ?? 'Object'								// some Objects do not have a constructor method
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)